libs/sdk-model/src/settings/index.ts

```diff
@@ -239,6 +239,11 @@ export interface ISettings {
      */
     enableSeparateTotalLabels?: boolean;
 
+    /**
+     * Enable theming on Panther
+     */
+    enablePantherTheming?: boolean;
```

Comment:
This is a strange name, why not just enableTheming?

Reply:
@ivan-nejezchleb decided to do so, but he is currently on PTO. I will switch to enableTheming, because I do not see it as a big deal of the difference, and he will decide later to switch it back if there is something huge about it.